src/App.js

```diff
@@ -1,23 +1,42 @@
-import logo from './logo.svg';
+
+import React,  { useEffect } from 'react'
 import './App.css';
+import { useDispatch, useSelector } from 'react-redux'
+import { loadDrinkData } from './redux/actions/liquorSearch.js'
+import { loadGameData } from './redux/actions/boardGame'
+import PairingForm from './Components/PairingForm';
+
+const App = () => {
+  const dispatch = useDispatch()
+  
+  const games = useSelector(state => state.boardGame.allGamesData.games)
+  
+  const getDrinkData = () => {
+    return fetch('https://www.thecocktaildb.com/api/json/v1/1/filter.php?c=Cocktail')
+      .then(res => res.json())
+  }
+
+  const getGameData = () => {
+    return fetch('https://api.boardgameatlas.com/api/search?name=Catan&pretty=true&client_id=JLBr5npPhV')
+      .then(res => res.json())
+  }
+    
+  const getData = () => {
+    Promise.all([getDrinkData(), getGameData()])
+      .then(data => {
+        dispatch(loadDrinkData(data[0]))
+        dispatch(loadGameData(data[1]))
+      })
+  }
+
+  useEffect(() => {
```

Comment:
I think we're all pleased to see this working example of the hook useEffect here - great call @raz-joan on trying to use it this way with the empty array to prevent an infinite loop.